.yarnrc

```diff
@@ -0,0 +1 @@
+workspaces-experimental true
```

Comment:
I *think* this isn't necessary anymore but I might be wrong

Reply:
I'll double check and we can remove if that's the case. 👍